### ISSUE TITLE
altering wsgi.py

### ISSUE THREAD

**Issue by bill-orange** [m0ngr31/kodi-alexa#108]
I would like to change a few of the responses to something a bit comical.

Instead of "rebooting system", it would say "a brain the size of a planet and that's the most important thing you can ask me to do, rebooting system"

Once revised, do I simply run again:

C:\Python27\Scripts\lambda-deploy.exe deploy

I want to be sure that the existing copy in Lambda is revised and not a second copy in Lambda created so maybe I am missing a step.

Thanks,

Bill

**Comment by bill-orange** [comment]
Never mind.  Answered my own question by reading the shutil wiki.  The command in my earlier post did the job.   

Snarky replies are added.  Much spousal eye rolling.

Bill

**Comment by digiltd** [comment]
You should ideally be using `python deploy-to-lambda.py` to prevent files being uploaded that don't need to be. Running `lambda-deploy deploy` will upload the entire folder `python deploy-to-lambda.py` will copy required files to a tmp folder, deploy from the tmp folder and then delete it for you.

**Comment by bill-orange** [comment]
I understand the improved efficiency but my command line stuff is a bit rusty.

While in my kodi-alexa-master folder, when I run python deploy-to-lambda.py, I get 'lambda-deploy' is not recognized as an internal or external command, operable program or batch file'.

I'm confused.  Should I copy lambda-deploy.exe from python27/scripts into my kodi-alexa-master folder so it can be found or will that create other problems?

Thanks,

bill - AB6OR

**Comment by digiltd** [comment]
I'm not sure about Python on Windows sorry. I expect you might need to add python to your system path or whatever the Windows equivalent is called.

So perhaps just stick to what you know works for now and forget I stuck my nose in :) 

But if you wanted to investigate: http://docs.python-guide.org/en/latest/starting/install/win/

**Comment by bill-orange** [comment]
Thanks,  The powershell command should do the trick.  However, what I ended up doing is editing deploy-to-lambda.py so that the path to 'lambda-deploy'  was included.  In other words, rather than invoking it by 'lambda-deploy.exe' I invoke it by 'C:\python27\scripts\lambda-deploy.exe'

I was reluctant to edit 'lambda-deploy' since I did not know how it worked.  The mops and the buckets are dancing enough already (Fantaisia reference)!
